File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main (void)
{
    int height = 0;
    do
    {
        printf("Height: ");
        height = get_int();
    } while (height < 0 || height > 23);


    for (int i = 2; i < height + 2; i++)
    {
        for (int k = height - i; k >= 0; k--)
        {
            printf(" ");
        }


        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: pset6/mario.py
"""
CS50

Problem set6

Mohammed Attya

"""

while True:
    n = input('Height: ')
    try:
        n = int(n, 10)
    except Exception as e:
        print('Retry: ...')
    else:
        break

if n > 13 or n < 1:
    print('Height must be a positive number and less than 13')
    exit(1)

for i in range(1, n+1):
    print(' ' * (n - i), end='')
    print('#' * (i + 1))
